src/Microsoft.Extensions.Options/OptionsServiceCollectionExtensions.cs

```diff
@@ -12,18 +10,37 @@ namespace Microsoft.Extensions.DependencyInjection
 {
     public static class OptionsServiceCollectionExtensions
     {
-        public static IServiceCollection AddOptions(this IServiceCollection services)
+        public static IServiceCollection AddOptionsMonitor<TOptions>(this IServiceCollection services) where TOptions : class, new()
         {
             if (services == null)
             {
                 throw new ArgumentNullException(nameof(services));
             }
 
-            services.TryAdd(ServiceDescriptor.Singleton(typeof(IOptions<>), typeof(OptionsManager<>)));
             services.TryAdd(ServiceDescriptor.Singleton(typeof(IOptionsMonitor<>), typeof(OptionsMonitor<>)));
             return services;
         }
 
+
+        public static IServiceCollection AddOptions<TOptions>(this IServiceCollection services) where TOptions : class, new()
+        {
+            if (services == null)
+            {
+                throw new ArgumentNullException(nameof(services));
+            }
+
+            services.AddSingleton(serviceProvider =>
```

Comment:
Well guessed. It could also be `AddSingletonOptions<TOptions>` and `TryAddSingletonOptions<TOptions>` to be closer to the service methods they correspond to. Though I'm guessing you wouldn't prefer that. :)